app/routes/account/benefits/plans/add.js

```diff
@@ -0,0 +1,14 @@
+import Route from '@ember/routing/route';
+import { inject as service } from '@ember/service';
+
+export default class PlanAddRoute extends Route {
+  @service store
+
+  async model () {
+    return this.store.createRecord('plan', { fromService: 'manual' });
```

Comment:
Removing this, don't want anything in there for a manual plan
```suggestion

  async model () {
    return this.store.createRecord('plan');
```